lms/djangoapps/teams/views.py

```diff
@@ -296,7 +296,14 @@ def get(self, request):
         if not queryset:
             return Response(status=status.HTTP_404_NOT_FOUND)
 
-        page = self.paginate_queryset(queryset)
+        # TODO: Remove this on migration to Django 1.8
```

Comment:
Nit: I was confused by the use of the word `migration` here, because we usually use it to talk about database migrations. I would use `update` instead.